packages/core/upload/server/controllers/content-api.js

```diff
@@ -127,4 +128,18 @@ module.exports = {
 
     await (id ? this.replaceFile : this.uploadFiles)(ctx);
   },
+
+  async proxy(ctx) {
+    const {
+      query: { fileKey },
+    } = ctx;
+
+    const url = await signFileUrl(fileKey);
+
+    if (!url) {
+      return ctx.notFound();
+    }
+
+    ctx.redirect(url); // Returns 302
+  },
```

Comment:
Do we want to remove this part for the content api now?

Reply:
yes!